src/python/doc/representations.rst

```diff
@@ -46,3 +46,24 @@ Metrics
    :members:
    :special-members:
    :show-inheritance:
+
+Basic example
+-------------
+
+This example computes the first two Landscapes associated to a persistence diagram with four points. The landscapes are evaluated on ten samples, leading to two vectors with ten coordinates each, that are eventually concatenated in order to produce a single vector representation.
+
+.. testcode::
+
+    import numpy as np
+    from gudhi.representations import Landscape
+    # A single diagram with 4 points
+    D = np.array([[0.,4.],[1.,2.],[3.,8.],[6.,8.]])
+    diags = [D]
+    l=Landscape(num_landscapes=2,resolution=10).fit_transform(diags)
+    print(l) 
+
+The output is:
+
+.. testoutput::
+
+    [[1.02851895 2.05703791 2.57129739 1.54277843 0.89995409 1.92847304 2.95699199 3.08555686 0.         0.64282435 0.         0.         0.51425948 0.         0.         0.        ]]
```

Comment:
On your development branch, the output I have is:
```python
[[0.         1.25707872 2.51415744 1.88561808 0.7856742  2.04275292
  3.29983165 2.51415744 1.25707872 0.         0.         0.
  0.31426968 0.         0.62853936 0.         0.         0.31426968
  1.25707872 0.        ]]
```
This may explain why `sphinx_py_test` fails in the test suite.